### PR TITLE
Remove node_modules from the final image

### PR DESCRIPTION
**Pull request by shouichi**

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/Dockerfile.tt [hunk 1]
```diff
@@ -78,6 +78,10 @@ RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
 
 <% end -%>
 
+<% if using_node? -%>
+RUN rm -rf node_modules
+<% end %>
```
Comment (akhilgkrishnan) [inline railties/lib/rails/generators/rails/app/templates/Dockerfile.tt:83]: can you move this under line 88?

Comment (shouichi) [reply railties/lib/rails/generators/rails/app/templates/Dockerfile.tt:83]: Thank you for the feedback. If I'm not mistaken, `node_module` does not exist in line 88. It is copied at line 93.

**Comment by byroot** [comment]
I think not including the source is a good practice, but the RUN here is in its own layer, so this making the image any smaller.

**Comment by shouichi** [comment]
Thank you for the review. If I'm not mistaken, it does reduce the image size. Because `node_modules` is removed from the build image and `node_module` is prevented from copying (in `COPY --from=build /rails /rails`) in the final image.

**Pull request merged**